README.md

```diff
@@ -106,30 +106,24 @@ If you plan to develop with TorchServe and change some of the source code, you m
     sudo apt-get install openjdk-11-jdk
     ```
 
-1. Install dependencies
-
-    ```bash
-    pip install psutil future
-    ```
-
 1. Clone the repo
 
     ```bash
     git clone https://github.com/pytorch/serve
     cd serve
     ```
 
-1. Make your changes executable
+1. Install from local path
 
     ```bash
-    pip install -e .
```

Comment:
Should we just add the manual build step here
```
python setup.py build
```
and then
```
pip install . # or pip install -U -e .
```
the `python setup.py build` should invoke [build in setup.py](https://github.com/pytorch/serve/blob/master/setup.py#L91) which should technically build the frontend and plugins and everything else :) .

Reply:
sounds like we should add this to scripts in PR #322 ?